Sources/CacheAdvance/EncodableInteger.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Created by Dan Federman on 12/3/19.
+//  Copyright © 2019 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+/// A protocol representing an integer that can be encoded as data.
+protocol EncodableInteger {
+
+    /// Converts the big-endian value in x to the current endian format and returns the resulting value.
+    static func swapBigIntToHost(_ x: Self) -> Self
+
+    /// The big-endian representation of self.
+    @inlinable var bigEndian: Self { get }
```

Comment:
Why add `@inlinable`?

Reply:
The property is `@inlinable` already on `UInt32` and `UInt64` (and likely other integer types). Figured staying consistent was valuable.